Makefile

```diff
@@ -93,16 +93,17 @@ build-js-po: node-version-check
 
 build: locale
 
-locale: build-js-po
+merge-locale-catalogs: build-js-po
+	$(PIP) install click babel
```

Comment:
`merge-catalogs` needs these

Reply:
It looks like babel's only needed for `bin/merge-catalogs` and `bin/find-good-catalogs`, yet it's present in `requirements-dev.txt` (might be able to get away with removing it from there). Were you trying to do this locally and did not have python dev dependencies installed? `click` is in `requirements-base.txt` though, so it's most likely something is trying to do this before sentry dev is installed...

Ideally `click` is also pinned the same as in `requirements-base.txt`. Or if we can make the assumption that sentry's installed, even better.

Reply:
Hmm I must not have had python dev dependencies installed then, not sure why it was unable to find `click` though. Will play around